otter/integration/tests/test_lbsh.py

```diff
@@ -155,8 +145,6 @@ def test_move_node_to_oob_lb(self):
         yield clb_as.delete_nodes(self.rcs, [the_node['id']])
         yield clb_other.add_nodes(self.rcs, [node_info])
 
-        nodes_as = yield clb_as.list_nodes(self.rcs)
-        nodes_other = yield clb_other.list_nodes(self.rcs)
         clb_as.wait_for_nodes(
```

Comment:
These need to be yielded, or the tests won't actually wait until this is the case.  :|